docs(react-grid): add a note about bug with grouping in Safari (#2675)
DevExpress/devextreme-reactive: packages/dx-react-grid/docs/guides/grouping.md

> The following example demonstrates remote grouping with local expanding/collapsing, as well as the approach described in the previous paragraph:

Reviewer: ```suggestion
The following example demonstrates remote grouping with local expanding/collapsing, and the approach described in the previous paragraph:
```

.embedded-demo({ "path": "grid-grouping/remote-grouping-with-local-expanding", "showThemeSelector": true })

> **Browser Support Notes:**

Reviewer 2: ```suggestion
## Browser Support Notes
```

> - Currently, virtual table scrolls when a column header is moved to a grouping panel. This behaviour is caused by a bug in the WebKit engine: [Unable to call event.preventDefault in dynamically added event listener](https://bugs.webkit.org/show_bug.cgi?id=184250).

Reviewer 2: ```suggestion
In Safari, the virtual table scrolls while a user is dragging a column header. This behavior is caused by a bug in WebKit: <a href="https://bugs.webkit.org/show_bug.cgi?id=184250" target="_blank">Unable to call event.preventDefault in dynamically added event listener</a>.
```

> In Safari, the virtual table scrolls while a user is dragging a column header. This behavior is caused by a bug in WebKit: <a href="https://bugs.webkit.org/show_bug.cgi?id=184250" target="_blank">Unable to call event.preventDefault in dynamically added event listener</a>.

Reviewer: ```suggestion
In Safari, the virtual table is scrolled when a user drags a column header. This is caused by a bug in WebKit: <a href="https://bugs.webkit.org/show_bug.cgi?id=184250" target="_blank">Unable to call event.preventDefault in dynamically added event listener</a>.
```
